edenai_apis/apis/google/tests/test_google_helpers.py

```
@@ -0,0 +1,186 @@
+            (50, 3.0),
+        ],
+    )
+    def test_volume_formatting(self, volume, expected):
+        assert get_formated_speaking_volume(volume) == pytest.approx(expected)
+
+
+class TestHandleGoogleCall:
+    @pytest.mark.unit
+    @pytest.mark.google
+    def test_success_returns_response(self):
+        func = lambda **kwargs: {"result": "ok"}
```

Ruff (0.15.10)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `func` as a `def`

(E731)